### PR TITLE
Percentof

### PR DESCRIPTION
**Pull request by puskin94**
@mintsoft @jagtalon 
Guys, sorry for the previous mess. I hope I fixed all the stuff, I really don't know how this happened but I think that all is currently rigth

### CODE REVIEW

**Comment by mintsoft** [comment]
@puskin94 looks like there's still a problem: https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/jobs/46496271#L1706

**Comment by puskin94** [comment]
@mintsoft It were only 2 lines of code that I had to delete ;) Now it should works! Please Ping me again If there are sill problems :+1:

File: lib/DDG/Goodie/PercentOf.pm [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+zci answer_type => "percent_of";
+zci is_cached   => 1;
+
+name "PercentOf";
+description "Makes Operations with percentuals";
+primary_example_queries "4-50%", "349*16%";
+
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/PercentOf/PercentOf.pm";
+attribution github => ["puskin94", "puskin"],
+            twitter => "twitterhandle";
```
Comment (jdorweiler) [inline lib/DDG/Goodie/PercentOf.pm]: can you change this to `twitter => [ "twitterhandle", "puskin"]`  or delete it if you would rather not have it ?  We're trying to clean up the attribution a bit.

**Comment by puskin94** [comment]
@jdorweiler Done!! ;)

**Comment by jdorweiler** [comment]
@puskin94 what about decimals?  If I search `4.23-50%` I get the result 11.5 so it's actually calculating `23-50%`

**Comment by puskin94** [comment]
@jdorweiler Thanks for the feedback and for the improvement idea! I just committed the fix!

**Comment by jdorweiler** [comment]
@puskin94 thanks,  Could you add some tests for those cases too.

**Comment by puskin94** [comment]
@jdorweiler Sure! Just updated now ;) Ah, I added the possibilty to use percentuals with decimals too ( as you can see in tests :+1:  )

**Comment by jdorweiler** [comment]
@puskin94 cool, looks good.  :+1:

**Comment by jagtalon** [comment]
@puskin94 it's live! https://duckduckgo.com/?q=4+-+50%25&ia=answer

**Comment by puskin94** [comment]
@jagtalon ahah in a week you merged like 3 of mine PR :) :+1:

**Pull request merged**